src/stratis_cli/_actions/_data.py

```diff
@@ -210,6 +198,30 @@
 MAXIMUM_DBUS_TIMEOUT_MS = 1073741823
 
 
+def _fetch_property(name, props, to_string):
```

Comment:
Please move this functionality into ```formatting.py``` as right now there is no necessity for stringifying these values other than for formatting.

Please also adapt the function so that it generates values suitable for printing in a columnar format when there is an error or the key is not found. You definitely don't want to print out the error from the daemon in the column; we can't guarantee that it is not enormous.

Ideally, you can parameterize the behavior for key not found or error just as you are already parameterizing for the value.

This would be easy to do with a subclass of dict, or alternatively a lighter-weight currying approach.

Reply:
P.S. There may be some documentation in the D-Bus API doc about how unknown, etc things are supposed to be printed out. If so, we should try to follow it, or if it seems wrong, revise it.

Reply:
I notice that in ```_stratisd_constants.py``` there seems to be a value "???".

Reply:
I've made a second pass at this.